crates/turborepo-ffi/src/lib.rs

```diff
@@ -444,14 +444,14 @@ pub extern "C" fn get_package_file_hashes_from_inputs(buffer: Buffer) -> Buffer
                 to_return.insert(filename, hash);
             }
             let file_hashes = proto::FileHashes { hashes: to_return };
-            let resp = proto::GetPackageFileHashesFromInputsResponse {
```

Comment:
Did you mean to include this change?

Reply:
Yes, since I changed the clippy code to error on any warnings, I had to fix this one